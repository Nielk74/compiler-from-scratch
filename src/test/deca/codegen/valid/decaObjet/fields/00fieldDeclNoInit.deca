// Description:
//    Programme avec une classe avec des champs de type de base (int, float, boolean)
//    et qui affiche ces champs (non initialis√©s)
//
// Resultats:
//    0
//    a.y == false
//    0.00000e+00
//
// Historique:
//    cree le 16/06/2022

class A {
    int x;
    boolean y;
    float z;
}
{
    A a = new A();
    println(a.x);
    if (!a.y) {
        println("a.y == false");
    } else {
        println("a.y == true");
    }
    println(a.z);
}