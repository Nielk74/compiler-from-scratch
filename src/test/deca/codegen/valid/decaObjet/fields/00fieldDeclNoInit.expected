0
a.y == false
0.00000e+00
